packages/theme/src/cli/flags.ts

```diff
@@ -31,3 +31,13 @@ export const themeFlags = {
     env: 'SHOPIFY_FLAG_ENVIRONMENT',
   }),
 }
+
+export const themeDevPreviewFlag = {
+  'dev-preview': Flags.boolean({
+    char: 'd',
```

Comment:
Considering [this guideline](https://github.com/Shopify/cli/blob/main/docs/cli-kit/command-guidelines.md#aliases--shortcuts-for-flags), I believe it would be a good idea to remove this alias.
```suggestion
```

(Note: We will likely need to regenerate manifests due to this change.)